### PR TITLE
Additional content for website

### PR DESCRIPTION
**Pull request by gnomeontherun**
* I renamed Web Components to Core Components, for consistency and simplicity
* I updated or added content in the get started and release sections
* Fixed a couple small formatting issues

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: apps/website/.vuepress/sidebar.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
```
Comment (hippee-lee) [inline apps/website/.vuepress/sidebar.js:2]: ⏳

**Review by hippee-lee — APPROVED** [review]

File: apps/website/.vuepress/theme/styles/overrides.scss [hunk 2]
```diff
@@ -327,9 +327,19 @@ body .no-clr-ui p {
...
 }
+
+// Fix footnotes styling
+.footnotes .footnote-item p {
+  display: contents;
+}
+.footnotes-sep {
+  margin: var(--cds-global-space-6) 0;
+  border: 0;
+  border-top: 1px solid var(--cds-token-color-neutral-900);
```
Comment (mathisscott) [inline apps/website/.vuepress/theme/styles/overrides.scss]: Wouldn't you want to use a design token for the border width?

**Comment by hippee-lee** [comment]
FYI - I have a fix for the website build in my PR that addresses the movement of get-started pages. 
We may want this PR to go in first as it will conflict with the other one...

**Review by mathisscott — APPROVED** [review]

File: apps/website/.vuepress/theme/styles/overrides.scss (hunk 2 above)
Comment (gnomeontherun) [reply apps/website/.vuepress/theme/styles/overrides.scss]: Because I wanted to see if you could find this.

**Review by mathisscott — APPROVED** [review]

File: apps/website/.vuepress/theme/styles/overrides.scss (hunk 2 above)
Comment (gnomeontherun) [reply apps/website/.vuepress/theme/styles/overrides.scss]: Fixed.

**Pull request merged**